pkg/common/bundleutil/unmarshal_test.go

```diff
@@ -4,23 +4,26 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/spiffe/go-spiffe/v2/bundle/spiffebundle"
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
 	"github.com/stretchr/testify/require"
 )
 
 func TestUnmarshal(t *testing.T) {
 	rootCA := createCACertificate(t)
 	trustDomain := spiffeid.RequireTrustDomainFromString("domain.test")
+	emptyBundle := spiffebundle.New(trustDomain)
+	emptyBundle.SetRefreshHint(0)
```

Comment:
why do you need to set hint? it will be nil as default, is that causing issues?

Reply:
@MarcosDY emptyBundle holds the expected value for the empty bundle ("{}"); this is just an adjustment for ensuring the refreshHint will be set to zero by default and not nil (to maintain the current behavior).